packages/jss-keyframes/src/index.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+import defaultJss, {SheetsRegistry, type Jss} from 'jss'
+import warning from 'warning'
+
+type Options = {
+  jss?: Jss,
+  name?: string
+}
+
+let counter = 0
+const registry = new SheetsRegistry()
+const stylesheets = new WeakMap()
+
+function getStyleSheet(jss: Jss) {
+  const existingStylesheet = stylesheets.get(jss)
+
+  if (existingStylesheet) {
+    return existingStylesheet
+  }
+
+  const stylesheet = jss.createStyleSheet({}, {meta: 'JSS Keyframes'})
```

Comment:
wouldn't it make sense to just have one style sheet?

Reply:
In case one wants to render that style sheet in a specific place in the head, like all jss style sheets, there should be a way to pass other options.

Reply:
yeah, the idea behind that was to support multiple versions of jss

Reply:
But I have an idea on how to actually allow the user to pass options to the sheet.

Reply:
Yes, its possible, but its still a complication.